engine/src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -7,16 +7,19 @@ import cromwell.engine.ExecutionStatus
 import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.workflow.BackendCallKey
+import slick.dbio.Effect.Read
+import slick.jdbc.GetResult
 import slick.profile.RelationalProfile.ColumnOption.Default
 import wdl4s.{NamespaceWithWorkflow, Scatter}
 
 import scala.language.postfixOps
-import scala.util.{Success, Failure, Try}
+import scala.util.{Failure, Success, Try}
 
 case class CallCacheHit(workflowId: String, callName: String)
 case class ExecutionWithCacheData(execution: Execution, cacheHit: Option[CallCacheHit])
 
-case class Execution(workflowExecutionId: Int,
+case class Execution(executionId: Option[Int] = None,
```

Comment:
Honestly I just blindly did an IntelliJ refactor because I thought it was necessary but it turns out it's not necessary.  I can change it back to the original signature and it'll also make the diff nicer